spec/emittr_spec.rb

```diff
@@ -103,6 +103,93 @@
     end
   end
 
+  describe '#on_any' do
+    let(:block)     { Proc.new {} }
+    let(:any_block) { Proc.new {} }
+
+    context 'when #emit is called' do
+      it 'calls #on_any blocks' do
+        emitter.on_any &any_block
+
+        emitter.on :first_any_test, &block
+        emitter.on :second_any_test, &block
+
+        expect(any_block).to receive(:call).with(:first_any_test).exactly(2).times
+        expect(any_block).to receive(:call).with(:second_any_test).exactly(2).times
+
+        2.times { emitter.emit :first_any_test }
+        2.times { emitter.emit :second_any_test }
+      end
+    end
+  end
+
+  describe '#off_any' do
+    let(:block) { Proc.new {} }
+
+    it 'removes listener from #any list' do
+      emitter.on(:off_any_test) { Proc.new {} }
+      listeners = emitter.send(:listeners)
+
+      emitter.on_any &block
+      emitter.emit :off_any_test
```

Comment:
it's better to these 2 expectations: one for the `call` and one for removing from listeners list